packages/svelte/src/internal/client/dom/elements/events.js

```diff
@@ -66,6 +66,24 @@ export function create_event(event_name, dom, handler, options) {
 	return target_handler;
 }
 
+/**
+ * Attaches a DOM event handler to an element and returns a function that detaches the event. The event handler
+ * will be processed through Svelte's internal event delegation system and is the preferred way to imperatively
+ * attach event handlers instead of using `addEventListener`.
+ *
+ * @param {Element} dom
+ * @param {string} event_name
+ * @param {EventListener} handler
+ * @param {AddEventListenerOptions} [options]
+ */
+export function attach(dom, event_name, handler, options = {}) {
+	var target_handler = create_event(event_name, dom, handler, options);
```

Comment:
`create_event` calls `addEventListener`, it's not using event delegation. What am I missing?

Reply:
It uses the target function which ensures any delegated events run as expected, whilst still attaching the event to the target manually.

Reply:
Ah, I see. The documentation is a bit misleading in that case, will rewrite it

Reply:
Sounds good.